goss-testing/tests/ncn/goss-validate-iuf-manifest.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# MIT License
+#
+# (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+{{ $pre_checks := .Vars.pre_checks }}
+{{ $scripts := .Env.GOSS_BASE | printf "%s/scripts" }}
+{{ $iuf_pre_checks := $scripts | printf "%s/python/iuf_pre_checks" }}
+{{ $validate_iuf_manifest := .Env.GOSS_BASE | printf "%s/scripts/python/validate_iuf_manifest.py" }}
+{{ $manifest := .Vars.iuf_manifest }}
+{{ $logrun := .Env.GOSS_BASE | printf "%s/scripts/log_run.sh" }}
+
+command:
+  {{ $testlabel := "iuf_validate_product_manifest" }}
+  {{$testlabel}}:
+    title: iuf validate product_manifest
+    meta:
+      desc: Validates iuf_product_manifest.yaml
+      sev: 0
+    exec: |-
+
+      # validate iuf product manifest
+      echo "Test case:validate_iuf_manifest"
```

Comment:
Why do you include this echo line? How are you running this Goss test where you are able to see that output?